.circleci/config.yml

```diff
@@ -57,6 +61,7 @@ commands:
           command: |
             cd sandbox
             bundle add rspec_junit_formatter --group test
+            <<#parameters.coverage>>export COVERAGE=true<</parameters.coverage>>
```

Comment:
Is that `<<#` an `if` statement on CircleCI :heart:? Do you have a reference for it? I looked for it, but I didn't find it.

Reply:
Yes, it's an `if`, and I have to search for it deep in the circleci docs every time, there's an example here, although somewhere there must be a place with a syntax reference: https://circleci.com/docs/reusing-config/#commands-usage-examples

Reply:
Oh, yeah, that's really hidden :slightly_smiling_face: Thanks for sharing!